lib/tasks/dev.rake

```diff
@@ -6,7 +6,24 @@ if Rails.env.development? || Rails.env.test?
     task prime: "db:setup" do
       include FactoryBot::Syntax::Methods
 
-      # create(:user, email: "user@example.com", password: "password")
+      create(:budget, user: User.create(
+        name: "admin",
+        email: "admin@example.com",
+        password: "secret",
+        admin: true
+      ))
+
+      9.times do
+        create(:budget, user: create(:user))
```

Comment:
The `:budget` factory creates a user by default, so this can we written:
`9.times { create(:budget) }`

Reply:
Done in sha: 7c2f098